lib/literal/enum.rb

```diff
@@ -149,6 +149,12 @@ def coerce(value)
 			case value
 			when self
 				value
+			when Symbol
+				begin
+					const_get(value)
+				rescue NameError
+					nil
```

Comment:
This should be `self[value]` I think to account for symbol typed Enums

Reply:
Good catch! I’ll write a test for that.